vision: add genesis principles (#1)
btrustteam/board-book: vision/genesis_principles.md

# ₿trust Genesis Principles

> ₿trust’s Genesis Principles are an immutable set of principles governing its operation and outlining its vision. They exist to guide the board and representatives of the trust, and to act as an accountability safeguard against bad actors.  

Reviewer: ```suggestion
₿trust’s Genesis Principles are an immutable set of principles governing its operation and outlining its vision. They exist to guide the board and representatives of the trust and will act as an accountability safeguard against bad actors.  
```

Author: Leaving as "to act" because "will" implies that they'll only do so at some (unspecified) point in the future?

Reviewer: Yes, that's true, '_to act_' is more definite and binding here.

> Board members will take ownership of this vision, and act in good faith to protect the interests and integrity of the organization. The specific implementation of these principles is the prerogative of the board and its selected representatives.

Reviewer: ```suggestion
Board members will take ownership of this vision and act in good faith to protect the interests and integrity of the organization. The specific implementation of these principles is the prerogative of the board and its selected representatives.
```

> 1. Bitcoin will provide a monetary system that enhances financial sovereignty for all. ₿trust will support Bitcoin exclusively. In the event of a hard fork it will follow the chain endorsed by the user majority.

Reviewer: ```suggestion
1. Bitcoin will provide a monetary system that enhances financial sovereignty for all. ₿trust will support Bitcoin exclusively. In the event of a hard fork, it will follow the chain endorsed by the user majority.
```

Author: Hm, do we need a comma here? My grammar is 0/10, but it _feels_ like it breaks the sentence strangely.

Reviewer: Yes, I believe the comma is needed here as "_In the event of a hard fork_" reads more like an introductory phrase.

> 2. ₿trust will maintain the highest levels of integrity. To that end, ₿trust does not exist to make a profit for itself or its representatives. The board will be mindful to prevent self-dealing within the organization and representatives will declare conflicts of interest as they arise.

Reviewer: ```suggestion
2. ₿trust will maintain the highest levels of integrity. To that end, ₿trust does not exist to make a profit for itself or its representatives. The board will be mindful to prevent self-dealing within the organization, and representatives will declare conflicts of interest as they arise.

```

> 3. Free and open source projects are fundamental to Bitcoin’s censorship resistance and resilience. ₿trust will focus its efforts on fostering developer talent to work on under-resourced free and open source Bitcoin projects. It will prioritize projects based on how dependent the bitcoin ecosystem is on them, and their capacity to accept new contributors. 

Author: Been thinking that we might want to move `under-resourced` into the prioritizing sentence, because we might want to support projects that aren't nescisarily under-resourced?

Suggested copy: 
`₿trust will focus its efforts on fostering developer talent to work on free and open source Bitcoin projects. It will prioritize projects based on how dependent the bitcoin ecosystem is on them, resource requirements and their capacity to accept new contributors.`

Reviewer: Good point! I think the suggestion is a better rewording to make the area of interest less restrictive. Additionally, we can chuck in the 'why' into your above suggestion, like so:

```suggestion
3. ₿trust will focus its efforts on fostering developer talent to work on free and open source Bitcoin projects, to maintain Bitcoin's censorship resistance and resilience. It will prioritize projects based on how dependent the bitcoin ecosystem is on them, resource requirements and their capacity to accept new contributors.
```

Author: This will break our pattern of starting with a "why" statement at the beginning of every principle, so I've left the "why" first here.

4. Transparency is critical to building trust and ensuring external accountability. Decisions will be made public within a reasonable time frame and financial statements will be published quarterly. Information that undermines representatives’ security and privacy will be exempt. The ₿trust team will engage with feedback that is given in good faith to the best of its ability. 

> 5. Decentralization of Bitcoin’s development improves its resilience, reach and relevance as a global money. ₿trust will operate in regions where Bitcoin development is uncommon.

Author: `as a global money` -> `as global money`?

Reviewer: ACK on that.
